packages/ng/simple-select/src/lib/input/select-input.component.html

```diff
@@ -4,7 +4,7 @@
 		<ng-container *luOptionOutlet="valueTpl || optionTpl; value: value"></ng-container>
 	</div>
 </div>
-<a *ngIf="clearable && isFilled" class="lu-select-clearer" role="button" (click)="clearValue($event)">
+<a *ngIf="clearable && isFilled" class="clear lu-select-clearer" role="button" (click)="clearValue($event)">
```

Comment:
Missing `href` attribute. We could consider using a `<div tabindex="0">` instead.

Reply:
We have larger issue here as the clear button is not even focusable.
Let's keep it out of scope and I'll open an issue.

Reply:
I think the missing `href` is why the `<a>` tag isn't focusable
https://codepen.io/marcfairbrother/pen/LYXyjNG